cg/meta/upload/mip/mip_rna.py

```diff
@@ -34,14 +34,16 @@ def upload(self, ctx: click.Context, case: Family, restart: bool) -> None:
 
         # Scout specific upload
         if DataDelivery.SCOUT in case.data_delivery:
```

Comment:
Maybe we can ask KSV if he would like files to be sent to Caesar if the scout upload fails? If no then we should switch the order around